cdc/capture/main_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package capture
+
+import (
+	"testing"
+
+	"go.uber.org/goleak"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
What scope does it verify? Package scope or repo scope?

Reply:
Package scope

Reply:
Is this leak test triggered by `go test`, and after we have migrated all tests to testify, can we remove the `leak_test` entry in makefile